test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -802,6 +802,500 @@ test.describe( 'Navigation block', () => {
 	} );
 } );
 
+test.describe( 'Navigation block - Frontend interactivity', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+		await requestUtils.deleteAllTemplates( 'wp_template_part' );
+	} );
+
+	test.beforeEach( async ( { requestUtils } ) => {
+		await Promise.all( [ requestUtils.deleteAllMenus() ] );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.deleteAllMenus(),
+			requestUtils.activateTheme( 'twentytwentyone' ),
+		] );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllPosts();
+		await requestUtils.deleteAllMenus();
+		await requestUtils.deleteAllTemplates( 'wp_template_part' );
+	} );
+
+	test.describe( 'Overlay menu', () => {
+		test.beforeEach( async ( { admin, editor, page, requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
```

Comment:
This can be removed before the theme got changed already in the upper test scope.